src/gsy_e/models/leaves.py

```diff
@@ -64,30 +62,23 @@
 
 scm_strategy_mapping = {
     LoadHoursStrategy: SCMLoadHoursStrategy,
-    DefinedLoadStrategy: SCMLoadProfile,
+    DefinedLoadStrategy: SCMLoadProfileStrategy,
     PVStrategy: SCMPVStrategy,
     PVPredefinedStrategy: SCMPVPredefinedStrategy,
     PVUserProfileStrategy: SCMPVUserProfile,
     StorageStrategy: SCMStorageStrategy
 }
```

Comment:
If I'm not wrong then it can be deleted, since it's obsolete.

Reply:
Unfortunately not this one, the one that can probably be removed since there are no plans to add it in the community datasheet is the SCMLoadHoursStrategy. Still though for the latter, I would like to maintain this because it allows the CLI user to set a constant consumption in a convenient way, without being obliged to use a profile.